example/src/main/scala/scalaz/example/FreeCoyoUsage.scala

```diff
@@ -1,95 +0,0 @@
-package scalaz.example
```

Comment:
Would be nice to see this reworked rather than deleted. People seemed to find it helpful.